cfpb/cfgov-refresh
# Breakpoints

Way, way, way back in #149, @himedlooff suggested some breakpoint labels based on our grid layout.

## Additions

- Added variables `tablet-max` and `desktop-min` to have more breakpoints set by variables

## Changes

- Replaced most breakpoints under 801px to a variable.
- Changed to mobile-first breakpoints where it made sense
- Refactored footer, header and bureau templates 

## Testing

There's a lot of changes here. I tested as I made changes, but if I were reviewing this again, I'd do the following.
- Check the header and footer
- Check the beta banner
- Visit an office page
- Visit /the-bureau/, /the-bureau-structure/
- Visit the contact-us page

## Review

- @anselmbradford 
- @sebworks 
- @jimmynotjim 

## Preview

[Preview this PR without the whitespace changes](?w=0)

## Notes

- Since [cf-layout](https://github.com/cfpb/cf-layout/blob/gh-pages/src/cf-layout.less#L1406) breaks on 801px instead of 800px, the breakpoint there needed to be moved there.
